app/src/main/res/layout/feeditemlist_item.xml

```diff
@@ -43,6 +43,12 @@
                     tools:src="@drawable/ic_drag_darktheme"
                     tools:background="@android:color/holo_green_dark"/>
 
+            <CheckBox
+                    android:id="@+id/selectCheckBox"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1" />
```

Comment:
What is the weight for? Weights are used when displaying multiple items in a single layout but we never actually want to display both drag handle and checkbox.